assets/scripts/dialogs/AnalyticsDialog.jsx

```diff
@@ -165,6 +181,19 @@ function AnalyticsDialog (props) {
               </p>
             </div>
             <div className="dialog-actions">
+              <label>
+                <input
+                  type="checkbox"
+                  checked={isVisible}
+                  onClick={toggleIsVisible}
+                  disabled={isToggleDisabled}
+                />
+                <FormattedMessage
+                  id="dialogs.analytics.toggle-visible"
+                  defaultMessage="Show analytics"
```

Comment:
Let's rename this label so it's clearer what is being shown and where. Maybe "Show capacity counts in segment labels" ?